src/org/labkey/test/util/UIContainerHelper.java

```diff
@@ -43,6 +43,12 @@ public void doCreateFolder(String parentPath, String child, String foldertype)
     @Override
     @LogMethod
     protected void doCreateProject(String projectName, String folderType)
+    {
+        doCreateProject(projectName, folderType, null);
+    }
+
+    @LogMethod
+    public void doCreateProject(String projectName, String folderType, String templateFolder)
```

Comment:
Make a separate method for creating from a template, rather than having to pass in a dummy folder type.
```suggestion
    {
        doCreateProject(projectName, folderType, null);
    }

    public void createProjectFromTemplate(String projectName, String templateFolder)
    {
        doCreateProject(projectName, "Create From Template Folder", templateFolder);
    }

    @LogMethod
    protected void doCreateProject(String projectName, String folderType, String templateFolder)
```

Reply:
Good suggestion. Done.